cmd/kube-controller-manager/app/controllermanager.go

```diff
@@ -104,10 +105,22 @@ func NewCMServer() *CMServer {
 		RegisterRetryCount:      10,
 		PodEvictionTimeout:      5 * time.Minute,
 		ClusterName:             "kubernetes",
+		VolumeConfig: VolumeConfig{
+			// default values here
+			PersistentVolumeRecyclerTimeoutNFS: 300,
```

Comment:
No, this should be specific to *a* config value.   A CLI flag has to map 1:1 to something in the binary which then gets set on an instance of VolumeConfig (with a more generic name) and passed to the plugin.

I am going to rename this top-level on to VolumeConfigFlags so as it avoid confusion with volume.VolumeConfig.